frontend/src/scenes/feature-flags/featureFlagLogic.ts

```diff
@@ -579,7 +585,32 @@ export const featureFlagLogic = kea<featureFlagLogicType>([
         projectsWithCurrentFlag: {
             __default: [] as Record<string, string>[],
             loadProjectsWithCurrentFlag: async () => {
-                return []
+                const orgId = values.currentOrganization?.id
+                const flagKey = values.featureFlag.key
+
+                const projects: ProjectsWithCurrentFlagResponse = await api.get(
+                    `api/organizations/${orgId}/feature_flags/${flagKey}`
+                )
+
+                // Put current project first
+                const currentProjectIdx = projects.findIndex((p) => p.team_id === values.currentTeamId)
+                if (currentProjectIdx) {
+                    const [currentProject] = projects.splice(currentProjectIdx, 1)
+                    const sortedProjects = [currentProject, ...projects]
+                    return sortedProjects
+                }
+                return projects
+            },
+        },
+        featureFlagCopy: {
+            copyFlag: async () => {
+                const orgId = values.currentOrganization?.id
+
+                return await api.create(`api/organizations/${orgId}/feature_flags/copy_flags`, {
```

Comment:
don't create the api path like so, instead add the copy_flags path here: https://github.com/PostHog/posthog/blob/master/frontend/src/lib/api.ts#L165

for an example see: https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/feature-flags/featureFlagLogic.ts#L484